src/xrScriptEngine/script_engine.cpp

```diff
@@ -786,9 +797,8 @@ void CScriptEngine::setup_callbacks()
 #if !XRAY_EXCEPTIONS
         luabind::set_error_callback(CScriptEngine::lua_error);
 #endif
-#ifndef MASTER_GOLD
```

Comment:
It would be silly not to try developing Open Xray to be as transparent as possible considering Stalker depends on mods and people's ability to make them. Logging should be as robust as possible in Release version.